api/types/access_request.go

```diff
@@ -228,7 +228,7 @@ func (r *AccessRequestV3) GetOriginalRoles() []string {
 }
 
 // getThreshold is a helper for getting a threshold by its stored index.
-func (r *AccessRequestV3) getThreshold(tid uint32) (AccessReviewThreshold, error) {
+func (r *AccessRequestV3) getThreshold(tid uint32) (AccessReviewThreshold, error) { //nolint:unused
```

Comment:
actually this was just an oversight on my part (stopped being used in a last-minute refactor).  feel free to delete.